AngularApp/src/assets/enabledResourceTypes.json

```diff
@@ -5,42 +5,48 @@
             "resourceType": "Microsoft.Web/sites",
             "templateFileName": "Detector_WebApp",
             "imgSrc": "assets/img/Azure-WebApps-Logo.png",
-            "versionPrefix": "v4/"
+            "versionPrefix": "v4/",
+            "azureCommImpactedServicesList": "appservice"
         },
         {
             "service": "AseService",
             "resourceType": "Microsoft.Web/hostingEnvironments",
             "templateFileName": "Detector_HostingEnvironment",
             "imgSrc": "assets/img/Azure-WebApps-Logo.png",
-            "versionPrefix": "v2/"
+            "versionPrefix": "v2/",
+            "azureCommImpactedServicesList": "appservice"
         },
         {
             "service": "ResourceService",
             "resourceType": "Microsoft.Logic/workflows",
             "templateFileName": "Detector_LogicApp",
             "imgSrc": "assets/img/Azure-LogicApps-Logo.png",
-            "versionPrefix": ""
+            "versionPrefix": "",
+            "azureCommImpactedServicesList": "logicapp"
         },
         {
             "service": "ResourceService",
             "resourceType": "Microsoft.ApiManagement/service",
             "templateFileName": "Detector_ApiManagementService",
             "imgSrc": "assets/img/Azure-ApiManagement-Logo.png",
-            "versionPrefix": ""
+            "versionPrefix": "",
+            "azureCommImpactedServicesList": ""
```

Comment:
For these resourceServices except webapp and logicapp there will be no need for the default value of azureCommImpactedServicesList?

Reply:
Right now, yes. Nothing will show for these resources.